### PR TITLE
Fixed debug script

### LINKED ISSUE #527: Debug publish script no longer functional
**Describe the bug**
Breakpoint features no longer functional on Debug publish builds

**To Reproduce**
1. Run debug publish script
2. Open in Unity 
3. Add break points to prefab test file
4. Attach debugger
5. Run scene
6. Attempt to reach break point

**Expected behavior**
Breakpoints stop execution and allow introspection

**Screenshots**
image

**Additional context**

### PR DESCRIPTION
**Pull request by RyRy79261**
closes #527

### CODE REVIEW

File: Packages/io.chainsafe.web3-unity.lootboxes/Chainlink/Runtime/Libraries/ChainSafe.Gaming.Lootboxes.Chainlink.dll
Comment (kantagara) [inline Packages/io.chainsafe.web3-unity.lootboxes/Chainlink/Runtime/Libraries/ChainSafe.Gaming.Lootboxes.Chainlink.dll:1]: did you intentionally delete those?

File: scripts/debug-publish-to-unity-package.bat [hunk 1]
```diff
@@ -1,30 +1,42 @@
...
 echo Restoring non-Unity packages...
 
-pushd ..\..
-dotnet restore
-popd
-
 echo Moving files to Unity package...
 
-pushd bin\debug\netstandard2.1\publish
+pushd bin\release\netstandard2.1\publish
```
Comment (kalambet) [inline scripts/debug-publish-to-unity-package.bat:18]: This is `debug-publish-to-unity-package.bat` probably this one should not be changed.

File: scripts/debug-publish-to-unity-package.sh [hunk 2]
```diff
@@ -12,19 +12,30 @@ dotnet publish -c debug -f netstandard2.1 /property:Unity=true
...
 echo Restoring non-Unity packages...
 
-pushd ../..
 dotnet restore
-popd
 
 echo Moving files to Unity package...
 
-cd bin/debug/netstandard2.1/publish
+pushd bin/release/netstandard2.1/publish
```
Comment (kalambet) [inline scripts/debug-publish-to-unity-package.sh:19]: Probably here too

File: Packages/io.chainsafe.web3-unity.lootboxes/Chainlink/Runtime/Libraries/ChainSafe.Gaming.Lootboxes.Chainlink.dll
Comment (RyRy79261) [reply Packages/io.chainsafe.web3-unity.lootboxes/Chainlink/Runtime/Libraries/ChainSafe.Gaming.Lootboxes.Chainlink.dll:1]: No, its just the result of the scripts running

**Review by kantagara — APPROVED** [review]

**Review by kantagara — APPROVED** [review]

**Review by robGG1997 — APPROVED** [review]

**Review by juans-chainsafe — APPROVED** [review]

File: scripts/debug-publish-to-unity-package.bat [hunk 3]
```diff
@@ -1,30 +1,42 @@
...
+)
+
+del Chainsafe.Gaming.Chainlink.dll
+del Chainsafe.Gaming.LootBoxes.Chainlink.dll
+
 del Microsoft.CSharp.dll
 if not exist ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries mkdir ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries\
 del ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries\* /F /Q
 copy *.dll ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries
-copy *.pdb ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries
```
Comment (kalambet) [inline scripts/debug-publish-to-unity-package.bat:26]: Why we don't need pdb? It keeps all the debug info

Comment (RyRy79261) [reply scripts/debug-publish-to-unity-package.bat:26]: @kalambet I just updated with the code from the main publish, will add back

File: scripts/debug-publish-to-unity-package.bat [hunk 4]
```diff
@@ -1,30 +1,42 @@
...
+    echo Directory does not exist, skipping actions.
+)
+
+del Chainsafe.Gaming.Chainlink.dll
+del Chainsafe.Gaming.LootBoxes.Chainlink.dll
+
 del Microsoft.CSharp.dll
 if not exist ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries mkdir ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries\
 del ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries\* /F /Q
 copy *.dll ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries
```
Comment (RyRy79261) [inline scripts/debug-publish-to-unity-package.bat:38]:
```suggestion
copy *.dll ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries
copy *.pdb ..\..\..\..\..\..\Packages\io.chainsafe.web3-unity\Runtime\Libraries
```

**Review by kalambet — APPROVED** [review]
LGTM!

**Pull request merged**